sous-chef/sous_chef/grocery_list/generate_grocery_list.py

```diff
@@ -90,15 +90,16 @@ def send_bean_preparation_to_todoist(self, todoist_helper: TodoistHelper):
         )
 
         # TODO change queue_bean_preparation into dataframe to use pandas here
-        for item in self.queue_bean_preparation:
-            label_list = item["group"].from_recipe + item["group"].from_day
-            todoist_helper.add_task_to_project(
-                task=self._format_bean_prep_task_str(item),
-                project=project_name,
-                label_list=label_list,
-                due_date=due_date,
-                priority=4,
-            )
+        if self.queue_bean_preparation:
```

Comment:
This could stay like that. [I hadn't realized that it's a public method that we call in the main. It wouldn't be ideal to test an attribute of the class outside of the class and then call a public method of that class based on that.]